Kitodo/src/test/java/org/kitodo/production/metadata/MetadataEditorIT.java

```diff
@@ -13,12 +13,15 @@
 
 import static org.awaitility.Awaitility.await;
 import static org.junit.Assert.assertTrue;
+import static org.kitodo.utils.ProcessTestUtils.META_XML;
+import static org.kitodo.utils.ProcessTestUtils.METADATA_BASE_DIR;
```

Comment:
Codacy issue
"Kitodo/src/test/java/org/kitodo/production/metadata/MetadataEditorIT.java#L17

Wrong order for 'org.kitodo.utils.ProcessTestUtils.METADATA_BASE_DIR' import."